presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriter.java

```diff
@@ -211,4 +218,18 @@ public String toString()
                 .add("writer", orcWriter)
                 .toString();
     }
+
+    private Page getOrcFileStatistics()
+    {
+        // OrcFileStatistics page layout:
+        //
+        // fileSize   rowCount
+        //  X             X
+        PageBuilder statsPageBuilder = new PageBuilder(Arrays.asList(BIGINT, BIGINT));
+        statsPageBuilder.declarePosition();
+        BIGINT.writeLong(statsPageBuilder.getBlockBuilder(0), ((Number) getWrittenBytes()).longValue());
```

Comment:
no need to cast to `Number` then use `longValue()`. The return type of getWrittenBytes() is already long. Same for the one below.